pages/glossary/web-page.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Web page
+key: web-page
```

Comment:
As far as I remember, we are still discussing around webpage definition, so we may not need this.